src/main/java/org/sagebionetworks/web/client/DisplayConstants.java

```diff
@@ -259,5 +259,9 @@
 	public static final String GO_TO_CURRENT_VERSION = "Go to the current version";
 	public static final String GO_TO_DRAFT_VERSION = "Go to the draft version";
 
+	public static final String AUTOCOMPLETE_ATTRIBUTE = "autocomplete";
+	public static final String AUTOCOMPLETE_VALUE_USERNAME = "username";
+	public static final String AUTOCOMPLETE_VALUE_CURRENT_PASSWORD = "current-password";
+	public static final String AUTOCOMPLETE_VALUE_NEW_PASSWORD = "new-password";
```

Comment:
Looks good.  Maybe add a link to the possible values for context? https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete